docs/changelog/2024.md

```diff
@@ -16,6 +16,23 @@
 .. role:: small
 ```
 
+## 2024-11-13 {small}`hub 0.30.1`
+
+### Enhancements
```

Comment:
Please take a brief look at the right sidebar and whether it helps to have these as subheadings or whether a simple paragraph (possibly boldened) looks better at least for the 2024 page where no other release has subheadings.

Reply:
Subsubheading